pkgs/development/interpreters/python/hooks/fix-python-deps-hook.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+from glob import iglob
+from shutil import copyfile
+from tempfile import NamedTemporaryFile
+
+import requirements
+
+
+requirements_txts = iglob(os.environ.get("requirementsTxt", "requirements*.txt"))
```

Comment:
Does that work with ``requirements.txt`` files in sub-directories?

Reply:
Not as is, but I think I can make it work.

But do we want to have this behavior? It may be somewhat surprising, no?

Reply:
It should be safe to do, so why not?

Some projects have dev or doc dependencies in sub some directories and it is annoying to define those files.

Also I didn't check the code but do we error if we do not find the dependency?

Reply:
Being typo proof is always a good idea

Reply:
Done.

Reply:
Oops, forgot the recursive search part. Leaving this unsolved for now.

Reply:
Done.